index.html

```diff
@@ -75,253 +83,361 @@ <h3><a href="#">LET'S CONNECT</a></h3>
       <section class="works-sect">
         <div class="projects">
           <div class="oneProject project1">
-            <div>
-              <a href="#"
-                ><img class="main-img" src="images/tonic_one.svg" alt="project"
-              /></a>
-              <h2><a href="#" class="tonic">Tonic</a></h2>
-            </div>
-            <div class="description">
-              <h5 class="descript"><a href="#">CANOPY</a></h5>
-              <ul>
-                <li>
-                  <a href="#"><img src="images/circle.svg" alt="circle" /></a>
-                </li>
-                <li><a href="#">Back End Dev</a></li>
-                <li>
-                  <a href="#"><img src="images/circle.svg" alt="circle" /></a>
-                </li>
-                <li><a href="#">2015</a></li>
-              </ul>
+            <div class="articles">
+              <article class="article-one">
+                <div class="project-img">
+                  <a href="#"
+                    ><img
+                      class="main-img"
+                      src="images/tonic_one.svg"
+                      alt="project"
+                  /></a>
+                  <a href="#"
+                    ><img
+                      class="desktop-img"
+                      src="images/desktop-image1.png"
+                      alt="project"
+                  /></a>
+                </div>
+              </article>
+              <article class="article-two">
+                <h2><a href="#" class="tonic">Tonic</a></h2>
+                <div class="description">
+                  <h5 class="descript"><a href="#">CANOPY</a></h5>
+                  <ul>
+                    <li>
+                      <a href="#"
+                        ><img src="images/circle.svg" alt="circle"
+                      /></a>
+                    </li>
+                    <li><a href="#">Back End Dev</a></li>
+                    <li>
+                      <a href="#"
+                        ><img src="images/circle.svg" alt="circle"
+                      /></a>
+                    </li>
+                    <li><a href="#">2015</a></li>
+                  </ul>
+                </div>
+                <div class="text-sect">
+                  <p class="text-description">
+                    A daily selection of privately personalized reads; no
+                    accounts <br class="breaker" />
+                    or sign-ups required.
+                  </p>
+                </div>
+                <div class="languages">
+                  <ul>
+                    <li><a href="#">html</a></li>
+                    <li><a href="#">css</a></li>
+                    <li><a href="#">javaScript</a></li>
+                  </ul>
+                </div>
+                <div>
+                  <button class="see_project">See Project</button>
+                </div>
+              </article>
```

Comment:
Please make the image size bigger as requested by the previous reviewer. It should be 50% of the card's width. It might have to do something with the dimensions set for this image. 😊
This is how it looks currently.
![image](https://user-images.githubusercontent.com/33268927/171800302-d1e97d08-fa91-41f9-b659-be1a2eabcd92.png)

Reply:
[muneebulrehman](https://github.com/muneebulrehman) thank you for your observation and feedback. I will adjust the image accordingly.